ibm/service/iampolicy/data_source_ibm_iam_action_control_assignment.go

```diff
@@ -397,61 +410,30 @@ func DataSourceIBMGetActionControlAssignmentActionControlAssignmentResourceCreat
 	return modelMap, nil
 }
 
-func DataSourceIBMGetActionControlAssignmentErrorResponseToMap(model *iampolicymanagementv1.ErrorResponse) (map[string]interface{}, error) {
+func dataSourceIBMGetActionControlAssignmentErrorResponseToMap(model *iampolicymanagementv1.AssignmentResourceError) (map[string]interface{}, error) {
```

Comment:
Will this 
`undefined: iampolicymanagementv1.AssignmentResourceError` be defined with latest go sdk change?